### DISCUSSION TITLE
Plugin Requests

### DISCUSSION THREAD

**Discussion by trendschau** [typemill/typemill#474] [Polls]
Do you have ideas or preferences for Typemill plugins? Simply upvote or ping me if you have ideas for other plugins. All plugins will be developed under a MAKER or BUSINESS license, so please make sure that you are ready to buy a licenses when you vote for a plugin.

**Comment by UlfurFenris** [comment]
Subscription and shop for digital goods (secure and easy, e.g. Snipcart) would be more important than ChatGPT gimmicks ;)

**Reply by trendschau** [reply]
Ok, there is a payhip plugin that I coded for a client, I could publish for maker-license if it fits to the use cases...

**Comment by derekderie** [comment]
I'm considering switching to Typemill from Wiki.js, but the lack of a SSO for my organization is holding me back. Adding this support would be very helpful to us.

**Reply by trendschau** [reply]
Hi derekderie, yes, that is indeed the most required feature right now and at the top of the list, however, I have a lack of time right now. I hope that I will find a way to work on it soon. 

Which SSO would be required in your case? LDAP, SAML?

**Comment by sohamakl** [comment]
I would like to see SAML which supports the 2FA option in addition.
Am 13. März 2026, 17:49 +0100 schrieb trendschau ***@***.***>:
